src/services/local-storage.service.js

```diff
@@ -8,7 +8,8 @@ export const clearLocalStorage = () => {
     language: DEFAULT_LANGUAGE,
     currency: DEFAULT_CURRENCY,
     cart: [],
-    theme: 'light'
+    theme: 'light',
+    countPerPage: 9
```

Comment:
Please, create the same constant as for default language and default currency

Reply:
fixed